locales/templates/app.pot

```diff
@@ -1123,3 +1167,44 @@ msgid ""
 "screenshots’ categories."
 msgstr ""
 
+#. Message shown in Home Screen with an icon. Users can click on the icon and
+#. click either they want this feature or not. We'll evaluate if we want to
+#. implement this feature base on the users' feedback.
```

Comment:
typos in the comment: 
* click -> it's normally a touch device. I'd say "choose if they either want…".
* base -> based.

Same notes on similar comments later.